package/MDAnalysis/analysis/msd.py

```diff
@@ -62,6 +62,14 @@
 Please cite [Calandri2011]_ [Buyl2018]_ if you use this module in addition to
 the normal MDAnalysis citations.
 
+.. warning::
+    To correctly compute the MSD using this analysis module, you must supply
+    coordinates in the **unwrapped** convention. That is, when they pass
```

Comment:
probably best to rephrase "they" to "atoms"?

Reply:
👍